ocs_ci/helpers/helpers.py

```diff
@@ -605,30 +605,39 @@ def create_ceph_block_pool(
     return cbp_obj
 
 
-def create_ceph_file_system(pool_name=None):
+def create_ceph_file_system(
+    cephfs_name=None, label=None, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
```

Comment:
Have we checked regression for this? i.e. Is there any TC which is consuming this function with pool_name param defined in the function call?

Reply:
Got clarity from Shivam that its validated and based on the output added Verified label.